catalog/gitops/cloudbuild-iam.yaml

```diff
@@ -11,20 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```

Comment:
nit: newline btw header and comment will improve readability

Reply:
iirc fix_lint.sh did this, either addlicense or kpt cfg fmt

Reply:
Yes, `gcr.io/kpt-fn/format:v0.1` removes the space between comment headers and the yaml. For now I'm going to leave as-is to avoid fighting the tooling.